src/uid2Sdk.ts

```diff
@@ -111,6 +124,11 @@ export class UID2 {
         const identity = this._opts.identity ? this._opts.identity : this._cookieManager.loadIdentityFromCookie();
         const validatedIdentity = this.validateAndSetIdentity(identity);
         if (validatedIdentity) this.triggerRefreshOrSetTimer(validatedIdentity);
+        if (this._opts.enableESP && !this._espEnabled) {
```

Comment:
Given that google has renamed ESP to SecureSignals, I think we should follow suit and stop using "ESP" in our codebase too.